lib/web_ui/lib/src/engine/window.dart

```diff
@@ -93,6 +97,10 @@ base class EngineFlutterView implements ui.FlutterView {
   late final PlatformViewMessageHandler platformViewMessageHandler =
       PlatformViewMessageHandler(platformViewsContainer: dom.platformViewsHost);
 
+  // TODO(goderbauer): Provide API to configure constraints. See also TODO in "render".
+  @override
+  ViewConstraints get physicalConstraints => ViewConstraints.tight(physicalSize);
```

Comment:
These physicalConstraints must come from the native embedder, and cannot be provided by the framework, right?

Reply:
Yeah, the embedder must tell me what would be legal sizes. If you just want to have it size to content in both dimensions, you could just provide min width/height = 0 and max width/height = double.infinity.

Reply:
This works. Once this gets rebased, I'll use the new JS API for the embedder to pass a Constraints object from JS into Flutter.